### PR TITLE
Add a new PhpUnitEndToEnd Fetcher

### PR DESCRIPTION
**Pull request by anaelChardan**
It adds the ability to launch PhpUnitEndToEnd test in a different tasks

### CODE REVIEW

**Comment by ahocquard** [comment]
What is the goal? As a lot of our phpunit are actually end tot ends?

**Comment by anaelChardan** [comment]
@ahocquard Some were in a folder EndToEnd if you look on this PR this folder https://github.com/akeneo/pim-enterprise-dev/tree/move-workorganization-test/tests/back/Pim/WorkOrganization/EndToEnd/Workflow Yes, I do agree that a lot of our Integration are actually end to end but I prefer to have them in a separate folder for now as those one are good. (you have written them :P)

**Comment by ahocquard** [comment]
Ok :+1:  maybe it's better to have a dedicated step to move and categorize tests later on (and then rewrite them).

File: vars/pimPhpunitEndToEndFetcher.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import net.sf.json.JSONArray
+import net.sf.json.JSONObject
+
+def call(Map params = [:]) {
+
+    String[] files = sh (returnStdout: true, script: 'find ' + params.paths.join(" ") + ' -name "*EndToEnd.php" -exec sh -c "grep -Ho \'function test\' {} | uniq"  \\; | sed "s/:function test//"').tokenize('\n')
```
Comment (ahocquard) [inline vars/pimPhpunitEndToEndFetcher.groovy]: be careful, tests with annotation will not be executed with that...

Comment (anaelChardan) [reply vars/pimPhpunitEndToEndFetcher.groovy]: It's the case for current Integration tests, what do you suggest ? :)

**Pull request merged**